src/Codeception/Module/AMQP.php

```diff
@@ -166,7 +167,7 @@ public function declareExchange(
         bool $auto_delete = true,
         bool $internal = false,
         bool $nowait = false,
-        ?array $arguments = null,
+        array|AMQPTable|null $arguments = null,
```

Comment:
@TavoNiievez Should this change being considered a BC break because it changes the type hint of a `public` method declared in a non-final class?

Reply:
But that is a decision I cannot make.
Just publish the next release by incrementing the major version ig